components/cam/src/physics/rrtmgp/radiation.F90

```diff
@@ -1373,6 +1382,25 @@ subroutine radiation_driver_sw(state, pbuf, cam_in, is_cmip6_volc, &
                          pint(1:nday,1:nlev_rad+1), &
                          col_indices=day_indices(1:nday))
 
+      ! Check temperatures to make sure they are within the bounds of the
+      ! absorption coefficient look-up tables
+      do iday = 1,nday
+         icol = day_indices(iday)
+         lat(iday) = state%lat(icol) * pi / 180._r8
```

Comment:
Have you checked this?  If state%lat is in radians isn't the conversion ( *180 / pi )

Reply:
Well duh, of course it is. Will push fix.